docs/integrations/sources/airtable.md

```diff
@@ -67,6 +103,7 @@ See information about rate limits [here](https://airtable.com/developers/web/api
 
 | Version | Date       | Pull Request                                             | Subject                                                |
 |:--------|:-----------|:---------------------------------------------------------|:-------------------------------------------------------|
+| 2.0.0   | 2023-01-27 | [21962](https://github.com/airbytehq/airbyte/pull/21962) | Added casting of native Airtable data types to JsonSchema types
```

Comment:
If OSS users need to also reset the schema and refresh their data, I think we need to indicate that here. I'm not sure if we have any documentation already about implications to end users of semver major version changes

Reply:
Not the best idea to put these descriptions inside of the changelog, but generally speaking, I agree. We should have a dedicated place or notifications that could guide the users through obstacles with upgrades/updates. 
I think we have enough information for the `set up guide` already, the rest - should be part of general documentation at least. There is nothing special about this one, the same rules as applied to the other connectors while upgrading and the changes are breaking.

Reply:
Hm, that's a good point. I'm not sure what the solution is to make sure that OSS users that don't get proactive outreach are aware